docs/src/administration/users.md

```diff
@@ -1,138 +1,162 @@
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
+## Environment types
 
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
+Each environment type groups one or more environments together so that you can manage access for all environments of a certain type.
+Platform.sh offers three types of environment groups: Production, Staging, and Development.
+You can assign user permissions for each environment type.
+Any permissions you assign to an environment type apply to all environments of that type.
 
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
+For example, if you assign user1 **Admin** permissions for Development environments, User1 effectively has **Admin** permissions to all development environments.
 
-## SSH access
+A few things to consider:
 
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
+* Only one environment per project can be the Production type. It's set automatically as the default branch and can't be overridden separately.
+* You can change an environment type and have multiple Staging or Development environments.
+* To customize who can use SSH, [set the access key](/configuration/app/access.html) in your platform.app.yaml
 
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
 
-## Manage user permissions in the console
+| Environment permission   | Description |
+|------------------------- |-------------|
+|Viewer                    | Viewers can view all environments of this type.|
+|Contributor               | Contributors can push code and branch all environments of this type. This role includes the Viewer permissions.
+|Admin                     | Admins can change settings and execute actions on all environments of this type. This role includes the Viewer and Contributor permissions.
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+## Manage users
 
-![Project user management screenshot](/images/management-console/settings-project-access.png)
+### Add a user to a project
 
-Selecting a user to edit that user's permissions or delete the user's access to the project entirely.
+To add a user to a project, follow these steps:
 
-To add a new user, click the `Add` button.
+{{< codetabs >}}
+---
+title="In the console"
```

Comment:
I'm sorry if my simplified example was misleading. It was intended to show the structure of the page, not the codetabs themselves. They currently require [more settings(https://github.com/platformsh/platformsh-docs/blob/master/contributing/markup-format.md#code-tabs) (`highlight=false` and `file=none`). I could change it to set those as the default, but that's why the deploy is currently failing.

This applies to all codetabs in here.